gcloud/datastore/__init__.py

```diff
@@ -48,7 +48,7 @@
 
 from gcloud import credentials
 from gcloud.datastore import _implicit_environ
-from gcloud.datastore.connection import Connection
+from gcloud.datastore import connection as connection_module
```

Comment:
What is the rationale for this change in pattern?  there isn't any clash with `Connection` that I can see.